src/phoenix/session/session.py

```diff
@@ -167,6 +168,27 @@ def url(self) -> str:
         """Returns the url for the phoenix app"""
         return _get_url(self.host, self.port, self.notebook_env)
 
+    def query_spans(
+        self,
+        *queries: SpanQuery,
+        start_time: Optional[datetime] = None,
+        stop_time: Optional[datetime] = None,
+        root_spans_only: Optional[bool] = None,
```

Comment:
Seems slightly inconsistent that this is outside?

Reply:
yea, let me remove it